service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/CaseManagementOrderReadyForJudgeReviewEventHandler.java

```diff
@@ -33,4 +37,19 @@ public void sendEmailForCaseManagementOrderReadyForJudgeReview(
         notificationService.sendEmail(CMO_READY_FOR_JUDGE_REVIEW_NOTIFICATION_TEMPLATE, email, parameters,
             eventData.getReference());
     }
+
+    @EventListener
+    public void sendEmailForCaseManagementOrderReadyForJudgeReviewToAllocatedJudge(
+        final CaseManagementOrderReadyForJudgeReviewEvent event) {
+        EventData eventData = new EventData(event);
+        CaseData caseData = mapper.convertValue(eventData.getCaseDetails().getData(), CaseData.class);
+
+        Map<String, Object> parameters = caseManagementOrderEmailContentProvider
+            .buildCMOReadyForJudgeReviewNotificationParameters(eventData.getCaseDetails());
+
+        String email = caseData.getAllocatedJudge().getJudgeEmailAddress();
```

Comment:
can we have `getAllocatedJudge ()` and/or `getJudgeEmailAddress()` being null at this point?

Reply:
Adding check as discussed in case of future